Makefile

```diff
@@ -62,8 +62,8 @@ build-xcframework-sample:
 # Building the .frameworsk.zip only works with Xcode 12, as there is no workaround yet for Xcode 13.
 build-framework:
 	@echo "--> Carthage: creating Sentry framework"
-	./scripts/carthage-xcode12-workaround.sh build --no-skip-current
-	./scripts/carthage-xcode12-workaround.sh archive Sentry --output Sentry.framework.zip
+	./scripts/carthage-xcode12-workaround.sh build --no-skip-current --use-xcframeworks
+	./scripts/carthage-xcode12-workaround.sh archive Sentry --use-xcframeworks --output Sentry.framework.zip
```

Comment:
`h`: This step should not use xcframeworks, why did you add this? It should create a `.framework.zip` and not a `.xcframework.zip`.